spec/features/collections/downloading_xml_spec.rb

```diff
@@ -26,16 +26,17 @@
         before do
           @file = "#{Rails.root}/#{@concept_id}.echo10"
           click_on 'ECHO 10'
-        end
 
-        after do
           # Seems to need a brief (>0.01) pause to actually find the file.
           sleep(0.1)
+        end
+
+        after do
           FileUtils.rm @file if File.exist?(@file)
         end
 
         it 'downloads the file' do
-          expect(File.exist?(@file))
+          expect(File.exist?(@file)).to eq(true)
```

Comment:
Why the extra .to eq(true) call?

Reply:
Without a matcher it isn't actually testing anything.  `expect(false)` passes.  It is unclear to me why it does not issue a syntax warning for this or what the use case there is for this behavior.